src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -113,14 +113,9 @@ const AppHeader: FunctionComponent<TProps> = ({
           </div>
           <div className="appHeader__nav">
             <div className="appHeader__nav_items">
-              <div className="appHeader__nav__item">{t("nav.pharmacy")}</div>
-              <div className="appHeader__nav__item">{t("nav.ward")}</div>
-              <div
-                className="appHeader__nav__item"
-                onClick={() => history.push("/billing")}
-              >
-                {t("nav.billing")}
-              </div>
+              <div className="appHeader__nav__item">{t("nav.patients")}</div>
```

Comment:
There's a missing redirect here. Clicking on this item should take the user to the main patient dashboard (the screen with "New Patient" and "Search Patient")

Reply:
@SteveGT96 any feedback on this?